src/components/graduation/CompletedCredits.tsx

```
@@ -0,0 +1,139 @@
+        (detail) => detail.id === summaryRow.id
+      );
+
+      return {
+        ...summaryRow,
+        rowType: "custom",
+        customRenderer: (
+          <div className="flex flex-col gap-3 p-2 border-t border-coolgray">
+            {detailSection ? (
+              detailSection.table.rows.map((course) => (
+                <SubjectCard {...course} />
+              ))
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)